docs/tests-github.md

```diff
@@ -0,0 +1,55 @@
+---
+category: DevelopInDepth
+
+---
+# Using GitHub Actions
+
+[Github Action](https://github.com/features/actions) makes it easy to automate all your software workflows, now with world-class CI/CD. Build, test, and deploy your code right from GitHub. Make code reviews, branch management, and issue triaging work the way you want.
+
+
+## GitHub Action Scripts
+
+All the GitHub Action build file are locate in `.github/workflows/*.yml`
+
+Each script is triggered by its own conditions, for more details see [Github Action Docs](https://docs.github.com/en/actions)
+
+Matomo uses GitHub Action to automatically run its build by various triggers.
+- [Build Tracker JS] Trigger by the comment `build js` into the pull request. That will compress js/piwik.js into matomo.js and piwik.js
+- [Build VUE] Auto trigger, it will build VUE file and commit back to the PR
```

Comment:
```suggestion
- [Build VUE] Triggered for PRs when commits are pushed. It will build VUE files and commit back to the branch or leave a comment, if the branch is not within the same repo.
```